packages/nodes-base/nodes/CrowdDev/CrowdDevNode.node.json

```diff
@@ -0,0 +1,18 @@
+{
+	"node": "n8n-nodes-base.crowdDev",
```

Comment:
That has to actually match with the name of the node. So is wrong right now, but will be correct once the node name got fied.